### ISSUE TITLE
Plot topomap

### ISSUE THREAD

**Issue by vferat** [mne-tools/mne-qt-browser#144]
**Is your feature request related to a problem? Please describe.**
When annotating bad segments and bad electrodes, it is often useful to have a look at the topography at a given timepoint (or the mean topography over several timepoints). It is not currently available in MNE, but we could take advantage of `mne-qt-browser` to implement it.

**Describe the solution you'd like**
Implement of new tool "plot topomap" which plots the topomap (`mne.viz.plot_topomap`) at the current timepoint (green line), for a selected time period , or the selected annotation in annotation mode. The general behavior would be similar to the interactive part of `raw.plot_psd` which allow to plot topomap of a selected frequency range.

The plot could be on an new window ( similar to the one used to plot channel locations) and update data automatically when the selected data changes.

**Describe alternatives you've considered**
Just a  button to create a new figure which plots topomap at current location (no automatic updating).

**Additional context**
The feature is currently available in Cartool:
![image](https://user-images.githubusercontent.com/28844486/191211931-48b3c97d-804b-4ee6-ad6e-72a3e2fa3b8f.png)

**Comment by larsoner** [comment]
Agreed we should have something like this. I think we should reuse the `evoked.plot(...)` code for this -- there is time-span-selection-to-topomap-plotting code that lives there already -- and it should work for mne-qt-browser and the matplotlib backends. I thought we discussed this already somewhere but I can't find it offhand...
